app/views/jobs/_form.html.erb

```diff
@@ -1,97 +1,162 @@
-                               </span>
-            </label>
+  </div>
+</section>
+
+<section class="vh-800" style="background-color: #2779E2;">
+  <div class="container h-1000">
+    <div class="row d-flex justify-content-center align-items-center h-1000">
+      <div class="col-xl-10">
+      <br>
+        <h1 class="text-white mb-4">Tell us about your organization</h1>
+        <div class="card" style="border-radius: 15px;">
+          <div class="card-body">
+            <div class="row align-items-center pt-4 pb-3">
+              <div class="col-md-3 ps-5">
+                <h6 class="mb-0">Employer Name</h6>
+              </div>
+              <div class="col-md-9 pe-5">
+              <%= f.text_field :job_author,  :class =>"form-control form-control-lg" , :placeholder=>"Enter your name"   %>
+              </div>
+            </div>
+            <hr class="mx-n3">
+            <div class="row align-items-center py-3">
+              <div class="col-md-3 ps-5">
+                <h6 class="mb-0">Logo Image</h6>
+              </div>
+              <div class="col-md-9 pe-5">
+                 <%= f.file_field :avatar,  :class=>"form-control form-control-lg" %>
+              </div>
+            </div>
+              <hr class="mx-n3">
+            <div class="row align-items-center py-3">
+              <div class="col-md-3 ps-5">
+                <h6 class="mb-0">Company URL</h6>
+              </div>
+              <div class="col-md-9 pe-5">
+              <%= f.text_field :url ,:class =>"form-control form-control-lg" , :placeholder=>"http://google.com"%>
+              </div>
+            </div>
+              <hr class="mx-n3">
+            <div class="row align-items-center py-3">
+              <div class="col-md-3 ps-5">
+                <h6 class="mb-0">Email</h6>
+              </div>
+              <div class="col-md-9 pe-5">
+              <%= f.text_field :email,:class =>"form-control form-control-lg" , :placeholder=>"email@gmail.com"%>
+              </div>
+            </div>
+              <hr class="mx-n3">
+            <div class="row align-items-center py-3">
+              <div class="col-md-3 ps-5">
+                <h6 class="mb-0">Twitter</h6>
+              </div>
+              <div class="col-md-9 pe-5">
+              <%= f.text_field :twitter,:class =>"form-control form-control-lg" , :placeholder=>"@user_name"%>
+              </div>
+            </div>
+          <hr class="mx-n3">
+            <div class="px-5 py-4">
+            <%= f.button :submit, :class=>"btn btn-primary btn-lg" %>
+            </div>
           </div>
         </div>
-      </div>
-      <div class="column">
-        <output id="list"></output>
+        <br>
+        <br>
       </div>
     </div>
-
-   <%= f.button :submit, class: "button is-primary is-large is-rounded mv4" %>
+  </div>
+</section>
+ 
 <% end %>
```

Comment:
why don't you break this file into multiple partials. This file seems to be really long.

Reply:
the file is divided in to its partial file